template.yaml

```diff
@@ -200,6 +200,19 @@ Resources:
               - s3:GetObject
             Resource: !Sub ${StructuredFieldsBucket.Arn}/*
 
+  DisableCloudWatchLoggingPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Deny
+            Action:
+              - logs:CreateLogGroup
```

Comment:
When this project is deployed to more accounts we shouldn't even see a log group for this lambda